src/index.html

```diff
@@ -1,17 +1,151 @@
 <!doctype html>
 <html lang="en">
+
   <head>
```

Comment:
There should be no blank lines between parent and child tags, fix it everywhere
```suggestion
<html lang="en">
  <head>
```

Reply:
Please pay attention to all comments: There should be no blank lines between parent and child tags, fix it everywhere